Lib/test/test_locale.py

```diff
@@ -574,6 +575,21 @@ def test_invalid_iterable_in_localetuple(self):
         with self.assertRaises(TypeError):
             locale.setlocale(locale.LC_ALL, (b'not', b'valid'))
 
+    def test_getfirstweekday(self):
+        self.assertEqual(locale.getfirstweekday(), 0)
```

Comment:
Wouldn't this test fail for users compiling and running tests with `fr_FR` as the default value for `LC_ALL`?

Reply:
Indeed I will use a BaseLocalizedTest class to ensure en_US is set.